packages/admin-ui/src/common/types.ts

```diff
@@ -429,15 +429,11 @@ export type ContainerState =
   | "exited" // exited A container that ran and completed("stopped" in other contexts, although a created container is technically also "stopped")
   | "dead"; // dead A container that the daemon tried and failed to stop(usually due to a busy device or resource used by the container)
 
-export type ChainDriver =
-  | "bitcoin"
-  | "ethereum"
-  | "ethereum2-beacon-chain-prysm"
-  | "monero";
+export type ChainDriver = "bitcoin" | "ethereum" | "ethereum2" | "monero";
 export const chainDrivers: ChainDriver[] = [
   "bitcoin",
   "ethereum",
-  "ethereum2-beacon-chain-prysm",
```

Comment:
Consider keeping this for backwards compatibility, don't break current versions of Prysm

Reply:
it is compatible with current prysm version, so we could just name it ethereum2